actor/v7pushaction/setup_deployment_information_for_push_plan.go

```diff
@@ -0,0 +1,13 @@
+package v7pushaction
+
+import "code.cloudfoundry.org/cli/api/cloudcontroller/ccv3/constant"
+
+func SetupDeploymentInformationForPushPlan(pushPlan PushPlan, overrides FlagOverrides) (PushPlan, error) {
```

Comment:
[suggestion / question / nonblocking] why is this separate from create_deployment_for_push_plan

Reply:
`actor.PreparePushPlanSequence` looked like the place where we were configuring the push plan by translating all the needed flags.

Reply:
Okay makes sense (the [pushplansequence](https://github.com/cloudfoundry/cli/blob/383393adc517d887f7c880fca9cb65bd9de07da0/actor/v7pushaction/actor.go#L70)). In that case what's strangely-shaped is the location (siblings within v7pushaction) of two deployment functions that both take pushPlans. I see that this parallels the situation for e.g. droplet config, so I'm happy to move on from this topic for now and leave it up to a separate refactor